src/js/smooch.jsx

```diff
@@ -196,6 +195,40 @@ export class Smooch {
         });
     }
 
+    monitorUrlChanges() {
+        const onhashchange = window.onhashchange;
+        const pushState = window.history.pushState;
+        const replaceState = window.history.replaceState;
+
+        window.onhashchange = function() {
```

Comment:
usually passing `arguments` works in some case but the real way of doing it would be to coerce it to an array (arguments is array-ish). Try wrapping it in `Array.from`to see if it works